scripts/directives/testing-type-specific-option.js

```diff
@@ -0,0 +1,19 @@
+function processNode(node, { _require, error }) {
```

Comment:
@cowboy This isn't particularly extensible just yet but wanted to get the bones in so we could chat through changes/updates.